examples/ex14.cpp

```diff
@@ -43,18 +43,33 @@ int main(int argc, char *argv[])
    const char *mesh_file = "../data/star.mesh";
    int ref_levels = -1;
    int order = 1;
+   bool rk = false;
```

Comment:
Please add two-three sample runs with kernelFEs for all examples, in the comments on the top.

Reply:
Done.